assets/js/translator.js

```diff
@@ -1,10 +1,14 @@
 /* @flow */
 
-var pairs = {}, chainedPairs = {}, originalPairs = pairs;
+var pairs = {}, chainedPairs = {}, originalPairs = pairs, chainedPaths = {}, weights = {};
 var srcLangs = [], dstLangs = [];
 var curSrcLang, curDstLang;
 var recentSrcLangs = [], recentDstLangs = [];
 var droppedFile;
+var curPaths = [], chosenPath = [];
+var svg, simulation, width = 800, height = 550, nodeSize = 20, nodeTextY = 5;
```

Comment:
It looks like all these should go a little lower as a set of constants with the rest.